contracts/ROSCA.sol

```diff
@@ -190,9 +195,9 @@ contract ROSCA {
       address[] members_,
       uint16 serviceFeeInThousandths_) {
     require(roundPeriodInSecs_ != 0 &&
-      startTime_ >= (now - MAXIMUM_TIME_PAST_SINCE_ROSCA_START_SECS) &&
+      startTime_ >= SafeMath.sub(now, MAXIMUM_TIME_PAST_SINCE_ROSCA_START_SECS) &&
       serviceFeeInThousandths_ <= MAX_FEE_IN_THOUSANDTHS &&
-      members_.length > 0);
+      members_.length > 1 && members_.length <= 256); // member count must be 1 < x < 256
```

Comment:
< 256 or <= 256? the code and comment are diffent

Reply:
done